app/controllers/auth-controller.js

```diff
@@ -0,0 +1,62 @@
+'use strict'
+
+const { validationResult } = require('express-validator/check')
+const jwt = require('jsonwebtoken')
+const userController = require('../controllers/user-controller')
+const validate = require('../utils/auth/validate')
+const logger = require('../utils/log/logger')
+const verify = require('../utils/auth/verify')
+
+exports.post_register = [
+    validate.validateUserRegistrationForm,
+    (request, response) => {
+        let errors = validationResult(request)
+        if (!errors.isEmpty()) {
+            errors.formatWith(({location, param, value, msg}) => msg) // removes all but the error message
+            response.status(400).send({status: 400, error: errors.array()})
+        } else {
+            const userData = {
+                username: request.body.username,
+                display_name: request.body.username,
+                email: request.body.email,
+                password: request.body.password
+            }
+            userController.createUser(userData).then((user) => {
+                console.log(process.env.SECRET)
```

Comment:
Instead of a just printing it out (which is a security risk) shouldn't you instead having handling logic? Something like `process.exit()` if it's not set? Since without it, your auth system shouldn't be functioning